components/island/theme.css

```diff
@@ -55,3 +55,34 @@
   border-color: var(--color-aqua);
   color: var(--color-aqua-darkest);
 }
+
+.segmented {
+  align-items: center;
+  border-bottom: 0;
+  border-top: 0;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: flex-start;
+  text-align: center;
+
+  &:not(:first-child),
+  &:not(:last-child) {
+    border-radius: 0;
+  }
+
+  &:not(:first-child) {
+    border-right: 0;
+  }
+
+  &:not(:last-child) {
+    border-left: 0;
+  }
+
+  &:first-child {
+    border-radius: 4px 0 0 4px;
```

Comment:
Can we move these instances of `4px` to a separate variable (and the one on L6 as well?) Thanks!